extmethods/tdma/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 2.8)
+
+set(EXT_TDMA true CACHE BOOL "EXT-TDMA: Build TDMA solver extension method.")
+if(NOT EXT_TDMA)
+    return()
+endif()
+
+include_directories(${CMAKE_SOURCE_DIR}/include)
+include_directories(${CMAKE_BINARY_DIR}/include)
+
+add_library(bh_tdma SHARED main.cpp)
+
+target_link_libraries(bh_tdma bh)
+set_target_properties(bh_tdma PROPERTIES COMPILE_FLAGS ${OpenMP_CXX_FLAGS} LINK_FLAGS ${OpenMP_CXX_FLAGS})
```

Comment:
You should properly only call `set_target_properties()` when `OPENMP_FOUND` is defined.
We might have to move [`find_package(OpenMP COMPONENTS bohrium)`](https://github.com/bh107/bohrium/blob/master/ve/openmp/CMakeLists.txt#L28) to the root cmake file?